lib/js/components/PopOver/PopOver.vue

```diff
@@ -272,6 +274,10 @@ export default {
 			type: String,
 			default: '',
 		},
+		shouldDisplay: {
+			type: Boolean,
+			default: true,
+		},
```

Comment:
```suggestion
		disabled: {
			type: Boolean,
			default: false,
		},
```

Reply:
or maybe we should add a new `POP_OVER_TRIGGER_ACTIONS` e.g. `none`?

Reply:
@tweetgeek using `none` is much more cleaner, please check again